metadata.rb

```diff
@@ -1,4 +1,4 @@
-# frozen_string_literal: true
+# frozen_string_literal: true ~FC066 ~FC069 ~FC078
```

Comment:
Can we look into ways to address these concerns that the linter raises without just ignoring them

Reply:
Okay, will still need to keep ~FC078 as it will require an OSI-approved open source license

Reply:
Done